src/ia-book-actions.js

```diff
@@ -478,21 +489,6 @@ export default class IABookActions extends LitElement {
     });
   }
 
-  async browseHasRenew() {
```

Comment:
moved this into main callback bc the lifecycle update connected to `this.lendingStatus` would interfere with sequential nature of main callback